week_1/5_apr/layout.css

```diff
@@ -0,0 +1,76 @@
+body {
+    height: 3000px;
+}
+
+.fixed {
+    position: fixed;
+    background-color: red;
+    width: 100%;
+    text-align: center;
+}
+
+.fixed .h1-fixed {
+    border: 8px solid wheat;
+    display: inline-block;
+    padding: 8px 12px;
+}
+
+.static {
+    position: static;
+}
+
+.relative {
+    position: relative;
+    top: 100px
+}
+
+.absolute {
+    position: absolute;
+    top: 50px;
+}
+
+.sticky {
```

Comment:
@codal-kcolaco when is sticky used?

Reply:
@abhi-codal Elements with sticky are positioned based on user's scroll position. They behave like a relative position element until they reach a specified scroll position where they become fixed. Suppose we have a nav bar after an image, and we want to make the nav bar as a fixed header when we scroll down the image, then we can use sticky where it uses relative positioning until we scroll down the image, then we make it fixed.